server_addon/houdini/client/ayon_houdini/api/hda_utils.py

```diff
@@ -418,3 +421,74 @@ def keep_background_images_linked(node, old_name):
 
     if changes:
         set_background_images(parent, images)
+
+
+def get_products_menu_items(node):
+    """Return products menu items
+
+    It gets a list of available products of the specified product types
+      within the specified folder path with in the specified project.
+
+    Users can specify those in the HDA parameters.
+
```

Comment:
```suggestion

    Args:
        node (hou.OpNode): The HDA node.

```

Reply:
solved in https://github.com/BigRoy/ayon-core/pull/9/commits/73a9833db2ac9d3621362995812788ae5d5ec46f